frontend/src/metabase/css/dashboard.module.css

```diff
@@ -218,3 +218,12 @@
   pointer-events: all;
   user-select: auto;
 }
+
+.ScalarValueContainer {
+  container-type: size;
+  container-name: scalar-value;
```

Comment:
nice that you added it! When I used container queries the last time I created a small bug because I didn't use